tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -564,6 +564,8 @@ public function testUpdateProject($data): array
     public function testUpdateProjectSMTP($data): array
     {
         $id = $data['projectId'];
+        
+        /**Test for SUCCESS: Valid Credentials*/
```

Comment:
Please format the comment like the other comments


```suggestion
        /**
         * Test for SUCCESS: Valid Credentials
         */
```